subsys/nrf_security/src/drivers/cracen/Kconfig

```
@@ -24,7 +24,7 @@ config CRACEN_LOAD_MICROCODE
 	  required when a bootloader has already loaded the microcode.
 
 config CRACEN_LIB_KMU
-	bool
+	bool "Enable CRACEN KMU library"
```

Shouldn't have `enable` in prompts:
```suggestion
	bool "CRACEN KMU library"
```